app/models/order_detail.rb

```diff
@@ -808,7 +808,7 @@ def can_reconcile_journaled?
   end
 
   def self.account_unreconciled(facility, account)
-    if account.is_a?(NufsAccount)
+    if account.class.using_journal?
```

Comment:
I would expect to be able to add `delegate :using_journal?, to: :class` in the `Account` object.  But it seems to not work with active support 3.2.22.1.
```
/Users/.../Workspace/txi/nucore/nucore-open/vendor/bundle/gems/activesupport-3.2.22.1/lib/active_support/core_ext/module/delegation.rb:139:in `module_eval': /Users/.../Workspace/txi/nucore/nucore-open/app/models/account.rb:42: syntax error, unexpected '.' (SyntaxError)
            class.__send__(:using_journal?, *ar...
                  ^
/Users/.../Workspace/txi/nucore/nucore-open/app/models/account.rb:44: syntax error, unexpected '.'
            if class.nil?                         ...
                     ^
/Users/.../Workspace/txi/nucore/nucore-open/app/models/account.rb:49: syntax error, unexpected keyword_end, expecting end-of-input
          end                                                 # end
```